containership/infrastructure-controller
# Update packages for Kubernetes 1.13.x

### Description

#### What does this pull request accomplish?

Update packages for Kubernetes 1.13.x

This includes a fix to make klog properly log to stderr instead of
crashing when trying to open a file.

See https://github.com/containership/cluster-manager/issues/42 for more
context.

#### What issue(s) does this fix?
* Closes #10

#### Additional Considerations

### Testing

Sanity test:

- Push changes as `mattkelly/infrastructure-controller:latest"
- Launch v1.13.4 HA cluster on DO
- Add `DISABLE_CLUSTER_MANAGEMENT_PLUGIN_SYNC=true` to coordinator deployment
- Update infra-controller to my tag
- Scale up master node pool by 1
- Scale down master node pool by 1
- See infra-controller properly removes etcd member

### Dependencies
* None
